config/config.go

```diff
@@ -24,7 +25,10 @@ func LoadFile(filename string) (*Config, error) {
 
 var (
 	DefaultModule = Module{
-		Version: 2,
+		Version:        2,
+		Retries:        3,
+		MaxRepititions: 25,
+		Timeout:        60,
```

Comment:
We should drop this to a second as it turns out to be a per-query timeout. Shouldn't this be a time.Duration?